### ISSUE TITLE
User notes are often displayed on wrong account in new modmail

### ISSUE THREAD

**Issue by dequeued0** [toolbox-team/reddit-moderator-toolbox#331]
This image should be self-explanatory: https://imgur.com/a/KSaNm5U.

This seems to consistently (or at least often) happen when first opening an archived conversation. Reloading the conversation will generally fix this.

**Comment by Geo1088** [comment]
Possibly related: https://discordapp.com/channels/535490452066009090/535490561478492161/732349096375091242
![image](https://user-images.githubusercontent.com/4165301/87508348-32742a80-c63d-11ea-9691-dc9851c6feca.png)

**Comment by creesch** [comment]
> Possibly related: https://discordapp.com/channels/535490452066009090/535490561478492161/732349096375091242
> ![image](https://user-images.githubusercontent.com/4165301/87508348-32742a80-c63d-11ea-9691-dc9851c6feca.png)

 
I don't think that is it as a page reload sometimes fixes it. Also we didn't move anything to the sidebar specifically as the buttons were always there. 

More likely it is a race condition where it simply tries to grab the information on page too early in some instances.

**Comment by Geo1088** [comment]
Potentially similar in root cause to the `getThingInfo` bug I fixed in #431. Need to double-check the elements being searched for modmail message information.

**Comment by creesch** [comment]
I believe I also did some work in this area recently and I can't remember when I ran into this the last time.
